apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -235,30 +229,24 @@ export const Subscriptions: FC<SubscriptionsProps> = ({
             ),
         });
 
-    const handleCriteriaChange = ({ page }: Criteria<Subscription>) => {
-        if (page) {
-            const { index, size } = page;
-            setPageSize(size);
-            setPageIndex(index * size);
-        }
-    };
-
     return (
-        <TableWithFilter
+        <TableWithFilter<Subscription>
```

Comment:
Adding the type is not needed I would say. TS can determine the type based on de props

Reply:
Can't hurt either right?

Reply:
True, but seeing this for example in plain text there might be a confusion if this is a <Subscription> component. Also I always understood that we should let typescript do its type-inference-thing as much as possible and only add types when TS is getting confused or when it is unclear.